src/test/java/racingcar/controller/PlayControllerTest.java

```diff
@@ -0,0 +1,40 @@
+package racingcar.controller;
+
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import racingcar.dto.RacingInput;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class PlayControllerTest {
+
+    @LocalServerPort
+    int port;
+
+    @BeforeEach
+    void setUp() {
+        RestAssured.port = port;
+    }
+
+    @DisplayName("POST PlayController Test")
+    @Test
+    public void playControllerTest() throws Exception {
+
+        RacingInput racingInput = new RacingInput("cyan,jade,bomi", 4);
+
+        RestAssured.given().log().all()
```

Comment:
인수테스트 해보셨군요!? 👏 

```suggestion
        ExtractableResponse<Response> response = RestAssured.given().log().all()...
```

시간이 되신다면 호출성공 뿐만아니라 반환된 데이터도 검증해보면 좋을듯하네요! ㅎ